src/XliffTasks/build/XliffTasks.targets

```diff
@@ -132,6 +136,26 @@
       <Output TaskParameter="Outputs" ItemName="%(Xlf.XlfOutputItem)" />
       <Output TaskParameter="Outputs" ItemName="FileWrites" />
     </GatherTranslatedSource>
+
+    <TransformTemplates Templates="@(VSTemplate)"
```

Comment:
To help with the naming confusion, consider filtering the VSTemplate items that have no resources here.
```
<ItemGroup>
  <_VSTemplateToLocalize Include="@(VSTemplate)" Condition="'%(VSTemplate.TranslatableResources)' != ''" />
</ItemGroup>
```

Then pass in only those, to have them all translated, and replace all of them on the way out.